src/Components/OATPropertyEditor/PropertyListItemNest.tsx

```diff
@@ -159,7 +159,7 @@ export const PropertyListItemNest = ({
             );
         } else if (
             newModel[propertiesKeyName][parentIndex].schema['@type'] ===
-            DTDLSchemaType.Object
+            'object'
```

Comment:
Should we be doing a case insensitive comparison here?

Does the DTDL spec require a specific casing? Seems weird we can't use the official enum here.